src/test/java/com/google/firebase/auth/FirebaseAuthIT.java

```diff
@@ -137,6 +137,83 @@ public void testDeleteNonExistingUser() throws Exception {
     }
   }
 
+  @Test
+  public void testDeleteUsers() throws Exception {
+    UserRecord user1 = newUserWithParams();
+    UserRecord user2 = newUserWithParams();
+    UserRecord user3 = newUserWithParams();
+
+    DeleteUsersResult deleteUsersResult = slowDeleteUsersAsync(
+        ImmutableList.of(user1.getUid(), user2.getUid(), user3.getUid())
+        ).get();
```

Comment:
Nit: Kind of strange line wrapping here.

Reply:
hmm... we should hook up clang-format or similar. I've ran clang-format over this file (just my changes; separately from the other review feedback).